src/components/BlockingViews/BlockingView.tsx

```diff
@@ -42,9 +63,19 @@ type BlockingViewProps = {
 
     /** Whether we should embed the link with subtitle */
     shouldEmbedLinkWithSubtitle?: boolean;
+
+    /** Style for the animation */
+    animationStyles?: StyleProp<ViewStyle>;
```

Comment:
should `animationStyles` and `animationWebStyle` be grouped such that they can only be provided alongside the `animation` prop, but not with the `icon` prop?